test/Microsoft.AspNetCore.Authentication.Test/OpenIdConnect/Infrastructure/TestDefaultValues.cs

```diff
@@ -0,0 +1,10 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+namespace Microsoft.AspNetCore.Authentication.Tests.OpenIdConnect.Infrastructre
```

Comment:
Also there already is a TestUtilities class, why can't you just put this in there instead of having a stand alone namespace + class + folder + file with one constant?

https://github.com/aspnet/Security/blob/dev/test/Microsoft.AspNetCore.Authentication.Test/OpenIdConnect/TestUtilities.cs#L14